### ISSUE TITLE
How To Ask for Salary Raise and Promotion?

### ISSUE THREAD

**Issue by thenoth7** [thenoth7/essaywriting#1]
While working in the same company for last many years, getting promoted to a stable position is your right. Moreover, increase in salary is also important. If your superior authorities are happy with you and your work then there should be no any hurdles in your way of getting promotion.

However, if your superiors are not taking interest in your promotion then its your turn to come up and take this serious matter into their consideration. While bringing this issue into spotlight, make sure you do it wisely. Make sure you are not leaving negative impressions on your boss while asking for promotion. This issue requires your complete attention and little planning would be beneficial for you. Following are some important tips, which will guide you a lot about raising this issue and driving it safely to its completion.

**Formal Appointment**

To raise the issue of your promotion and pay rise before your boss, get a formal appointment with your supervisor. Arranging a formal meeting with you boss will alert him/her that you have something important to discuss with them. By this way you will get more attention.

**Prepare Yourself**

Before attention the formal meeting with your boss, make sure you are well prepared to face your boss. Meeting your boss without enough preparations will ruin your impression and that could put you in critical situation and terms with your boss. Make sure you have practiced to talk about all the issues and points.

**Keep Control on Your Nerves**

While dealing with your boss regarding your salary raise issue, make sure you are not getting hyper. Try to stay calm and don’t get nervous. Such mistakes can be mistaken by your boss. He might think that you need the promotion desperately and your boss can use this weakness. Avoid being emotional.

**Be What You Are**

Bear in mind that your main purpose is only to convince your boss about raising your salary not to get the job. Be what you are if you want things move in the directions that could benefit you.

**State the Reason Clearly**

It’s not necessary that you prepare a long speech or give long detailing about your achievements to your boss. Just focus on the main reason, which urges you to demand an increase in your salary. Try to involve your boss into conversation instead of delivering long speech about why they should raise your salary.

**Wait for a Reply**

Give your boss a good time and meanwhile be patient. Thank your boss for his time and tell him that you will wait for his kind decision. Give him enough time to analyze your productivity, performance and value to the firm. Don’t expect the answer at the end of the meeting or next day.

**Avoid Giving Ultimatums**

You cannot pressurize your boss by telling them that you will quit the job if they won’t increase your salary etc. Bear in mind, there is a great number of unemployed people, who are ready to take your position even in lower compensation. Giving ultimatum to your boss will not be beneficial for you, so avoid it.

**Carry out a Market Research**

It’s better and wise to carry out a market search in order to find the pay of your position in other forms. Your efforts for the company also hold value towards your salary raise issue. Make sure you are going to quote expected salary that is not too much high or not too low.

In short, all above tips should help you to raise the issue of promotion and salary raise before your boss. Always remember that a little planning and nice way of presenting your say could benefit you a lot.

Lynette Bobbitt has PhD in psychology and social studies. Also Lynette has experience in HR management more 5 years. Lynette writes articles in free time for [essay writing companies](https://tutoriage.com/) Tutoriage.com.
